Yank/yamlbuild.py

```diff
@@ -1177,14 +1178,17 @@ class YamlBuilder:
     def yank_options(self):
         return self._isolate_yank_options(self.options)
 
-    def __init__(self, yaml_source=None):
+    def __init__(self, yaml_source=None, platform_name=None):
```

Comment:
That's exactly why I added it as part of the constructor and not part of the parsed options.